### DISCUSSION TITLE
How to translate Docsearch component and modal ?

### DISCUSSION THREAD

**Discussion by Zogsha** [vuejs/vitepress#1368] [Q&A] [answered]
Hi, thanks for your work on vitepress!

My documentation site is written in French, so I need the Docsearch component in French.

I have two problems :

1. There is a `algolia.buttonText` option in `themeConfig`. I set it to "Rechercher" instead of the default "Search", but whenever I click on the search component or use "CTRL+K", the placeholder turns back to "Search". It only displays "Rechercher" when the page refreshes. I only tested in dev.

2. In the Docsearch API Reference, there is `translations` option (https://docsearch.algolia.com/docs/api#translations). I can't figure out how to configure it.

I think __1.__ is not intended, but I didn't look further. Is __2.__ intended ? If that is the case, do you plan to let us translate the Doscsearch modal via `algolia` config in `themeConfig`?

Thanks,
Aliosha

**Comment by brc-dd** [comment, marked as answer]
I've created an issue from this (#1381). We will be fixing it in #1339 itself.
